drivers/ethernet/eth_xlnx_gem.c

```diff
@@ -529,22 +524,22 @@ static int eth_xlnx_gem_start_device(const struct device *dev)
 	dev_data->started = true;
 
 	/* Disable & clear all the MAC interrupts */
-	sys_write32(ETH_XLNX_GEM_IXR_ALL_MASK,
+	sys_write32(ETH_XLNX_GEM_IDRCLR_MASK,
 		    dev_conf->base_addr + ETH_XLNX_GEM_IDR_OFFSET);
-	sys_write32(ETH_XLNX_GEM_IXR_ALL_MASK,
+	sys_write32(ETH_XLNX_GEM_IDRCLR_MASK,
 		    dev_conf->base_addr + ETH_XLNX_GEM_ISR_OFFSET);
 
 	/* Clear RX & TX status registers */
-	sys_write32(0xFFFFFFFF, dev_conf->base_addr + ETH_XLNX_GEM_TXSR_OFFSET);
-	sys_write32(0xFFFFFFFF, dev_conf->base_addr + ETH_XLNX_GEM_RXSR_OFFSET);
+	sys_write32(~0, dev_conf->base_addr + ETH_XLNX_GEM_TXSR_OFFSET);
+	sys_write32(~0, dev_conf->base_addr + ETH_XLNX_GEM_RXSR_OFFSET);
```

Comment:
same deal

```suggestion
	sys_write32(~0U, dev_conf->base_addr + ETH_XLNX_GEM_TXSR_OFFSET);
	sys_write32(~0U, dev_conf->base_addr + ETH_XLNX_GEM_RXSR_OFFSET);
```

but, IMHO, the initial `0xffffffff` seems to be more straight-forward to me and this change seems unnecessary.

Reply:
Yeah, that change was probably unnecessary. No real benefit regarding formatting / readability, no technical benefit. I've reverted them.